content/pages/sponsorships.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Sponsorships
+slug: /sponsorships/
+---
+
+If you are interested in sponsoring please send a mail to <a href="mailto:sponsors@rustfest.eu">sponsors@rustfest.eu</a>.
+
+---
+
+A fundamental part of RustFest has always been sponsorships, as RustFest strives for low ticket prices and a detailed program supporting underrepresented people (e.g. BIPoC, LGBTQIA+ and/or disabled members of the Rust community), making a point to be as welcoming and safe as possible for everyone.
```

Comment:
...and a detailed program supporting underrepresented people (e.g. BIPoC, LGBTQIA+ and/or disabled members of the Rust community) **as well as people who otherwise couldn't afford,** making a point to be as welcoming and safe as possible for everyone.

Reply:
Fixed.